app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Admin::UsersController < ApplicationController
+  before_action :require_admin
+
+  def index
+    @users = User.all
+  end
+
+  def show
+    if !User.all.pluck(:id).include?(params[:id].to_i)
```

Comment:
This is there so if the user is deleted from the user show page, the admin can be redirected back to the user index